assets/stylesheets/overall.scss

```diff
@@ -1,6 +1,6 @@
 body {
     font-family: $default-font;
-    line-height: 1.428571429;
+    line-height: normal;
```

Comment:
If that fixes something why was it set to that specific number in before? Did you crosscheck that it still looks fine even when there is no custom_links page adding the second line?

Reply:
![image](https://github.com/os-autoinst/openQA/assets/1204189/7a434d2b-105c-45ec-844c-984a66aaf67f)

This is what I'm seeing

Reply:
no it doesnt. I will have to revert it